source/PlayerInfo.h

```diff
@@ -247,6 +248,9 @@ class PlayerInfo {
 	
 	
 private:
+	// Internal implementation of BuyShip and GiftShip
+	void ReceiveShip(const Ship *model, const std::string &name, bool gifting);
+	
```

Comment:
Perhaps move this to above or below SelectShip since both functions relate to ships. I'd also change the comment to match the one in the .cpp file.